### DISCUSSION TITLE
Add some custom snippets

### DISCUSSION THREAD

**Discussion by anotherhadi** [LazyVim/LazyVim#1311] [Q&A] [answered]
Hi!
I want to add a custom snippet for a specific file type (md)
I do also want "auto-focus" on the snippet to quickly change some of the text included in the snippet (Like the "table" snippet in markdown)

I have no idea how to achieve this inside LazyVim, so I need your help!

**Comment by radvil** [comment]
You can do this within LuaSnip config, just follow the official doc

**Comment by AsterismH** [comment, marked as answer]
I have the same problem and found the following link quite helpful:https://github.com/LazyVim/LazyVim/discussions/54#discussioncomment-4675882.
